vulnerabilities/helpers.py

```diff
@@ -233,12 +216,55 @@ def get_item(object: dict, *attributes):
     >>> assert(get_item({'a': {'b': {'c': 'd'}}}, 'a', 'b', 'e')) == None
     """
     if not object:
-        LOGGER.error(f"Object is empty: {object}")
+        logger.error(f"Object is empty: {object}")
```

Comment:
Let's return from here only as discussed and remove the log
```suggestion
        return
```

Reply:
Done.